### PR TITLE
Update Nos\Orm_Behaviour_Tree to order results

### PR DESCRIPTION
**Pull request by SwithFr**
Auto order results if Positionable behavior detected

### CODE REVIEW

File: framework/classes/orm/behaviour/tree.php [hunk 1]
```diff
@@ -177,7 +177,14 @@ public function find_children_recursive($item, $include_self = true)
     protected static function _populate_id_children($current_item, $children_relation, &$array)
     {
         $pk = \Arr::get($current_item->primary_key(), 0);
-        foreach ($current_item->get($children_relation) as $child) {
+        $params = array();
+	    $isPositionnable = $current_item::behaviours('Novius\Positionable\Orm_Behaviour_Positionable');
```
Comment (Meroje) [inline framework/classes/orm/behaviour/tree.php]: indentation looks odd

**Comment by Meroje** [comment]
:+1:

File: framework/classes/orm/behaviour/tree.php (hunk 1 above)
Comment (jguyomard) [inline framework/classes/orm/behaviour/tree.php]: Please indent by 4 spaces

**Comment by jguyomard** [comment]
Please indent by 4 spaces, not tab! 
:-1:

**Comment by shaoshiva** [comment]
Too much code style change.

**Comment by jguyomard** [comment]
imo, adding a reference to an unofficial application (novius_positionable) here seems to be a bad idea.

**Pull request closed without merging**